src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -211,22 +199,23 @@ public bool TryGetValue(object key, out object result)
             ValidateCacheKey(key);
             CheckDisposed();
 
-            DateTimeOffset utcNow = _options.Clock.UtcNow;
+            DateTime utcNow = UtcNow;
 
-            if (_entries.TryGetValue(key, out CacheEntry entry))
+            if (_entries.TryGetValue(key, out CacheEntry tmp))
```

Comment:
Why is this change necessary?

Reply:
The out variable can't be enregistered - using a separate local will mean less memory accesses.